down

```diff
@@ -0,0 +1,9 @@
+set +x 
```

Comment:
I think this script overlaps too much with wipe. I say delete wipe (wipe wipe ;-). 

Also, I think both of these changes should be documented in the README